src/consts.js

```diff
@@ -1,35 +1,97 @@
-import openshift from './components/openshift.svg';
-import hcm from './components/hcm.svg';
-import insights from './components/insights.svg';
-import rhelCs from './components/rhel-cs.svg';
+import { CloudCircleIcon, CloudServerIcon, ChartSpikeIcon, OpenshiftIcon } from '@patternfly/react-icons';
 
 export const activeTechnologies = [
     {
+        entitlement: 'insights',
         name: 'insights',
         id: 'insights',
         url: 'insights/actions',
-        icon: insights,
+        icon: ChartSpikeIcon,
+        iconProps: {
+            height: '3.5em',
+            width: '3.5em'
+        },
         title: 'Red Hat Insights',
+        emptyTitle: 'Get Started with Red Hat Insights',
+        emptyText: 'With predictive analytics, avoid problems and unplanned downtime in your \
```

Comment:
Let's not use breaks in these, just style it to have a max-width. If a person is using a large/small screen, let the viewport figure out where to break the text.

Reply:
These are just to break JS line so eslint would not complain.

Reply:
Beautiful